torchvision/__init__.py

```diff
@@ -3,6 +3,7 @@
 from torchvision import transforms
 from torchvision import utils
 
+__version__ = '0.1.9'
```

Comment:
What do you think about letting the version be automatically generated, as done by PyTorch? The relevant parts of the code can be found in https://github.com/pytorch/pytorch/blob/master/setup.py#L157-L173 and https://github.com/pytorch/pytorch/blob/master/setup.py#L591-L601

Reply:
I've seen this, but I thought it is too complicated for complete python project like torchvision.

Reply:
Looks like the current pattern is used by pip itself, so figured should be good enough for us too.